src/main/kotlin/com/sourcegraph/cody/edit/sessions/DocumentCodeSession.kt

```diff
@@ -4,15 +4,15 @@ import com.intellij.openapi.diagnostic.Logger
 import com.intellij.openapi.editor.Editor
 import com.intellij.openapi.project.Project
 import com.sourcegraph.cody.agent.CodyAgent
-import com.sourcegraph.cody.agent.protocol.EditTask
+import com.sourcegraph.cody.agent.protocol_generated.EditTask
 import com.sourcegraph.cody.edit.FixupService
 import java.util.concurrent.CompletableFuture
 
 class DocumentCodeSession(
     controller: FixupService,
     editor: Editor,
     project: Project,
-) : FixupSession(controller, project, editor) {
+) : FixupSessionV1(controller, project, editor) {
```

Comment:
does it work with v2?

Reply:
I've not tried any of the other systems with FixupSessionV2. I think neither of them is the final setup as there's some other problems that need addressing (sync points primarily), I just didn't want to make changes that would break existing functionality.